<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#startButton {
background-color:rgba(255,255,255,120);
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:rgb(245, 183, 179);
font:Bold 14px;
padding:4px;
}
QPushButton#startButton:hover {
background-color: rgb(255,255,255,60);
border-style: inset;
}
QPushButton#startButton:pressed {
 background-color: rgb(186, 139, 136);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="categoryButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#categoryButton {
background-color:rgba(255,255,255,120);
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:rgb(245, 183, 179);
font:Bold 14px;
padding:4px;
}
QPushButton#categoryButton:hover {
background-color: rgb(255,255,255,60);
border-style: inset;
}
QPushButton#categoryButton:pressed {
 background-color: rgb(186, 139, 136);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#exitButton {
background-color:rgba(255,255,255,120);
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:rgb(245, 183, 179);
padding:4px;
}
QPushButton#exitButton:hover {
background-color: rgb(255,255,255,60);
border-style: inset;
}
QPushButton#exitButton:pressed {
 background-color: rgb(186, 139, 136);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>671</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../.designer/backup/background.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>startButton</zorder>
   <zorder>categoryButton</zorder>
   <zorder>exitButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
